packages/base/src/util/FocusableElements.ts

```diff
@@ -36,6 +36,17 @@ const isElemFocusable = (el: HTMLElement) => {
 	return el.hasAttribute("data-ui5-focus-redirect") || !isElementHidden(el);
 };
 
+const isUI5ElementWithNegativeTabIndex = (el: HTMLElement) => {
```

Comment:
why do we need this?

Reply:
This covers `<ui5-input tabindex="-1"></ui5-input>`, which is used in the FileUploader component.